ci-operator/jobs/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main-presubmits.yaml

```diff
@@ -0,0 +1,72 @@
+presubmits:
+  konflux-workspaces/workspaces:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build03
+    context: ci/prow/e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-konflux-workspaces-workspaces-main-e2e
+    rerun_command: /test e2e
+    run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
```

Comment:
should we also look for changes in `^e2e`?

Reply:
Fixed in da9f87d5.